server/src/services/dataService.py

```diff
@@ -1,20 +1,12 @@
 import datetime
 import pandas as pd
-import sqlalchemy as db
-from sqlalchemy.orm import sessionmaker
 from .databaseOrm import Ingest as Request
+from utils.database import db
 
 
 class DataService(object):
```

Comment:
We can probably remove this constructor altogether at this point
We can move the table name to a constant in this class and the indv methods can define alternate tables (when and if) we decide to break the db into diff tables

Reply:
Good call, I removed the constructor. And then updated the services that call the DataService, so it's created without arguments.

As for the table name, turns out it wasn't even being used anywhere in the DataService. So I deleted it for now. If we split up tables we can add it as a constant, so it would be like a default table that can be overridden.